cmd/sidecars/network-passt-binding/callback/callback.go

```diff
@@ -49,6 +49,20 @@ func OnDefineDomain(domainXML []byte, domSpecMutator DomainSpecMutator) ([]byte,
 		return nil, err
 	}
 
+	const sharedMemoryBackingAccessMode = "shared"
+	if updatedDomainSpec.MemoryBacking == nil {
+		updatedDomainSpec.MemoryBacking = &domainschema.MemoryBacking{}
+	} else {
+		if updatedDomainSpec.MemoryBacking.Access != nil && updatedDomainSpec.MemoryBacking.Access.Mode != sharedMemoryBackingAccessMode {
+			return nil, fmt.Errorf("memory backing access mode must be 'shared'; cannot override existing mode: %s",
+				*updatedDomainSpec.MemoryBacking.Access)
+		}
+	}
+
+	updatedDomainSpec.MemoryBacking.Access = &domainschema.MemoryBackingAccess{
+		Mode: sharedMemoryBackingAccessMode,
+	}
+
```

Comment:
This check seems like it should be done by Mutate method, any specific reason its done here? 

Question, why is it necessary to care for verifying the memory backing access mode is shared, is it expected to be configured by other entities?
If not, it seems redandemt IMO. At least this is the current approach in the code base.